TLM/TLM/State/Keybinds/KeybindSetting.cs

```diff
@@ -0,0 +1,114 @@
+using JetBrains.Annotations;
+using UnityEngine;
+
+namespace TrafficManager.State.Keybinds {
+    using ColossalFramework;
```

Comment:
Actually it is StyleCop warning SA1200. If we accept making StyleCop happy on this warning, then more usings will move inside, rather than staying outside.
![image](https://user-images.githubusercontent.com/288724/60534570-6a465380-9d02-11e9-9fd8-5bf86500a3a8.png)

Reply:
It's certainly more semantic to have them inside the namespace. Although personally I find it easier to read them outside. It's not really an issue for now, I guess once harmony patch is done we'll be in better position to to a mass code cleanup so can address this when that happens.